scripts/test-harness-contracts.ps1

```
@@ -2694,6 +2694,47 @@ Check-Match '.claude/skills/pelizzai-review/SKILL.md' 'Splitting the task review
+# The set itself is part of the contract: dropping a file from the table (or duplicating one to
+# shadow another) must fail, not silently shrink the coverage.
+$expectedHotPathPaths = @(
+    '.claude/skills/pelizzai-core/SKILL.md'
+    '.claude/skills/pelizzai-router/SKILL.md'
+    '.claude/skills/pelizzai-preferences/SKILL.md'
+    '.claude/skills/pelizzai-review/SKILL.md'
+    '.claude/skills/pelizzai-execute/SKILL.md'
+    '.claude/skills/pelizzai-execute/references/task-cycle.md'
+)
+$configuredHotPathPaths = @($hotPathBudgets | ForEach-Object { $_.Path })
+Check ((@($configuredHotPathPaths | Sort-Object -Unique).Count -eq $expectedHotPathPaths.Count) -and (-not (Compare-Object -ReferenceObject ($expectedHotPathPaths | Sort-Object) -DifferenceObject ($configuredHotPathPaths | Sort-Object)))) 'hot-path budget table covers exactly the six expected files, no duplicates' "configured: $($configuredHotPathPaths -join ', ')"
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)